lib/puloon/puloon-dispenser.js

```diff
@@ -52,7 +51,11 @@ BillDispenser.prototype.reset = function reset (cassettes) {
 }
 
 BillDispenser.prototype.dispense = function dispense (notes) {
-  return this.device.dispense(notes)
+  try {
+    return this.device.dispense(notes)
+  } catch (err) {
+    return Promise.reject(err)
+  }
```

Comment:
surronded with try catch so we're sure to return a promise

Reply:
Better to make device.dispense return a proper promise, instead.

Reply:
Better to make device.dispense return a proper promise, instead.